tests/console/rsync.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: This module creates files that are synced afterwards using rsync.
+# Maintainer: Ciprian Cret <ccret@suse.com>
+
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils;
+use version_utils qw(is_opensuse is_sle is_jeos);
+
+sub run {
+    #create the folders and files that will be synced
+    select_console 'root-console';
+    assert_script_run('mkdir /tmp/rsync_test_folder_a');
+    assert_script_run('mkdir /tmp/rsync_test_folder_b');
+    assert_script_run('echo rsync_test > /tmp/rsync_test_folder_a/rsync_test_file');
+    assert_script_run("echo '#!/bin/sh\\necho Hello World' > /tmp/rsync_test_folder_a/rsync_test_sh.sh");
+    assert_script_run('tar -cvf /tmp/rsync_test_folder_a/rsync_test_tar.tar /tmp/rsync_test_folder_a/rsync_test_file');
+
+    my $md5_initial_file = script_output('md5sum /tmp/rsync_test_folder_a/rsync_test_file');
+    my $md5_initial_sh   = script_output('md5sum /tmp/rsync_test_folder_a/rsync_test_sh.sh');
+    my $md5_initial_tar  = script_output('md5sum /tmp/rsync_test_folder_a/rsync_test_tar.tar');
+    #in case localhost is already inside known_hosts
+    if (script_run '! test -e ~/.ssh') {
+        assert_script_run('ssh-keygen -R localhost');
+    }
+    script_run('rsync -avzr /tmp/rsync_test_folder_a/ root@localhost:/tmp/rsync_test_folder_b', 0);
+    # assert_screen 'remote-ssh-login';
+    if ((is_jeos) || (is_sle)) {
```

Comment:
Brackets around `is_...` subs are not necessary, do it rather the way you do it in `elsif`.